plugins/Installation/templates/_systemCheckSection.twig

```diff
@@ -1,4 +1,8 @@
 {% import _self as local %}
+<script type="text/javascript" src="plugins/Morpheus/javascripts/piwikHelper.js"></script>
```

Comment:
@geekdenz have you tried adding it to `Installation` controller in the `getInstallationJs` method? Does that work? That's the JS that is loaded.

Reply:
Yeah that is better, thank you for the hint. At the time I wasn't quite sure where that code was, so created a draft anyway.